test/functional/terminal/tui_spec.lua

```diff
@@ -229,6 +229,38 @@ describe('TUI', function()
     ]])
   end)
 
+  it('interprets <Esc>[27u as <Esc>', function()
```

Comment:
```suggestion
  it('interprets <Esc>[27u as <Esc> #17198', function()
```

Reply:
This behavior is not introduced in this PR. It is just the behavior of libtermkey.